src/components/OpeningHour.jsx

```diff
@@ -35,12 +35,24 @@ const OpeningHour = ({ schedule, showNextOpenOnly = false }) => {
     </div>;
   }
 
+  const NextTransition = () => {
+    if (!nextTransition || showNextOpenOnly && status !== 'closed') {
+      return null;
+    }
+
+    const text = status === 'close'
+      ? 'reopening at {nextTransitionTime}'
+      : 'until {nextTransitionTime}';
+
+    return ` - ${_(text,
```

Comment:
Using a variable here will prevent `xgettext` from extracting the translation keys automatically.

Reply:
True, I updated